CMake/README.md

```diff
@@ -0,0 +1,68 @@
+
+## Abseil CMake build instructions
+
+
+### Recommended usage : incorporate Abseil into an  CMake project
+
+  For API / ABI compatibility reasons, it is recommended to build
+  and use abseil in a subdirectory of your project or as an embedded
+  dependency
+
+  This is similar to the recommended usage of the googletest framework
+  ( https://github.com/google/googletest/blob/master/googletest/README.md )
+
+  Build and use step-by-step
+
+
+  1- Download abseil and copy it in a sub-directory in your project.
+      or add abseil as a git-submodule in your project
+
+  2- If not done yet, download and copy in your project the two dependencies of
+      abseil `cctz` and `googletest`
+
+    * cctz  https://github.com/google/cctz
+    * googletest https://github.com/google/googletest
+
+  3- You can then use the cmake command `add_subdirectory()` to include
+  abseil directly and use the abseil targets in your project.
+
+    Note: Abseil requires CCTZ and the googletest framework. Consequently,
+    the targets  `gtest`, `gtest_main`, `gmock` and `cctz` need
```

Comment:
In its current proposal, cctz would be better referred as `cctz::cctz`.

Reply:
Since the cctz PR will likely undergo some change, I consider this non-blocking